### PR TITLE
Update CASK_LANGUAGE_REFERENCE.md

### PR DESCRIPTION
**Pull request by miccal**
I could not find a current example of a Cask using `plist_set(key, value)` in the `*flight Mini-DSL` section.

### CODE REVIEW

File: doc/CASK_LANGUAGE_REFERENCE.md [hunk 1]
```diff
@@ -34,26 +33,40 @@ Each Cask is a Ruby block, beginning with a special header line. The Cask defini
 
 ```ruby
 cask 'alfred' do
```
Comment (vitorgalvao) [inline doc/CASK_LANGUAGE_REFERENCE.md]: This cask is just an example. We’re not supposed to update it every time.

File: doc/CASK_LANGUAGE_REFERENCE.md [hunk 2]
```diff
@@ -67,10 +80,10 @@ Each of the following stanzas is required for every Cask.
 | ------------------ |------------------------------ | ----------- |
 | `version`          | no                            | application version; give value of `:latest`  if versioned downloads are not offered
 | `sha256`           | no                            | SHA-256 checksum of the file downloaded from `url`, calculated by the command `shasum -a 256 <file>`. Can be suppressed by using the special value `:no_check`. (see also [Checksum Stanza Details](#checksum-stanza-details))
-| `url`              | no                            | URL to the `.dmg`/`.zip`/`.tgz`/`.tbz2` file that contains the application. A [comment](#when-url-and-homepage-hostnames-differ-add-a-comment) should be added if the hostnames in the `url` and `homepage` stanzas differ (see also [URL Stanza Details](#url-stanza-details))
+| `url`              | no                            | URL to the `.dmg`/`.zip`/`.tgz`/`.tbz2` file that contains the application. A [comment](#when-url-and-homepage-hostnames-differ-add-a-comment) should be added if the hostnames in the `url` and `homepage` stanzas differ. (see also [URL Stanza Details](#url-stanza-details))
```
Comment (vitorgalvao) [inline doc/CASK_LANGUAGE_REFERENCE.md]: Period shouldn’t be there.

File: doc/CASK_LANGUAGE_REFERENCE.md [hunk 3]
```diff
@@ -181,7 +192,7 @@ There are currently some arbitrary limitations on Cask tokens which are in the p
 
 Its first instance should use the latin alphabet, include the software vendor’s name, and be as verbose as possible while still making sense.
 
-A good example is [`pycharm-ce`](https://github.com/caskroom/homebrew-cask/blob/fc05c0353aebb28e40db72faba04b82ca832d11a/Casks/pycharm-ce.rb#L6#L7). `Jetbrains PyCharm Community Edition` makes sense even though it is likely never referenced as such anywhere, but `Jetbrains PyCharm Community Edition CE` doesn’t, hence why it has a second line. Another example are casks whose original names do not use the latin alphabet, like [`cave-story`](https://github.com/caskroom/homebrew-cask/blob/0fe48607f5656e4f1de58c6884945378b7e6f960/Casks/cave-story.rb#L7#L9).
```
Comment (vitorgalvao) [inline doc/CASK_LANGUAGE_REFERENCE.md]: This example still stands, and must not be removed.

File: doc/CASK_LANGUAGE_REFERENCE.md [hunk 4]
```diff
@@ -282,16 +293,16 @@ When the hostnames of `url` and `homepage` differ, the discrepancy should be doc
 # URL_HOSTNAME is the official download host per the vendor homepage
 ```
 
-Examples can be seen in [visit.rb](https://github.com/caskroom/homebrew-cask/blob/cafcd7cf7922022ea607c5811c63d45863c7ed36/Casks/visit.rb#L5) and [vistrails.rb](https://github.com/caskroom/homebrew-cask/blob/cafcd7cf7922022ea607c5811c63d45863c7ed36/Casks/vistrails.rb#L5).
```
Comment (vitorgalvao) [inline doc/CASK_LANGUAGE_REFERENCE.md]: This must remain as it was. Linking to a specific revision means the link will always point to what we want.

File: doc/CASK_LANGUAGE_REFERENCE.md [hunk 5]
```diff
@@ -282,16 +293,16 @@ When the hostnames of `url` and `homepage` differ, the discrepancy should be doc
...
 
 ### Difficulty Finding a URL
 
-Web browsers may obscure the direct `url` download location for a variety of reasons. Homebrew-cask supplies a script which can read extended file attributes to extract the actual source URL for most files downloaded by a browser on OS X. The script usually emits multiple candidate URLs; you may have to test each of them:
+Web browsers may obscure the direct `url` download location for a variety of reasons. Homebrew-Cask supplies a script which can read extended file attributes to extract the actual source URL for most files downloaded by a browser on OS X. The script usually emits multiple candidate URLs; you may have to test each of them:
 
 ```bash
-$ ./developer/bin/list_url_attributes_on_file <file>
+$ /usr/local/Library/Taps/caskroom/homebrew-cask/developer/bin/list_url_attributes_on_file <file>
```
Comment (vitorgalvao) [inline doc/CASK_LANGUAGE_REFERENCE.md]: There’s no guarantee the path is this one. Change instead to `$(brew --repository)/Library/Taps/caskroom/homebrew-cask/developer/bin/list_url_attributes_on_file <file>`.

File: doc/CASK_LANGUAGE_REFERENCE.md [hunk 6]
```diff
@@ -314,7 +325,7 @@ The value of the `appcast` stanza is a string, holding the URL for an appcast wh
 | ------------------ | ----------- |
 | `:sha256`          | a string holding the SHA-256 checksum of the most recent appcast which matches the current Cask versioning
 
-Example: [atom.rb](../../127387b9fc686370ffa92c01eeed8979df9e1621/Casks/atom.rb#L7#L8)
```
Comment (vitorgalvao) [inline doc/CASK_LANGUAGE_REFERENCE.md]: This must remain as it was. Linking to a specific revision means the link will always point to what we want.

File: doc/CASK_LANGUAGE_REFERENCE.md [hunk 7]
```diff
@@ -330,7 +341,7 @@ Note that `brew cask search` and `brew cask list` are not yet capable of using t
 
 Cask authors should use the most specific license category which is also correct. Generic categories are provided for difficult cases. `:unknown` is also perfectly fine if you are unsure.
 
-Example: [Chromium](http://www.chromium.org/chromium-os/licenses) includes code with multiple licenses, all of which are open source. Chromium licensing is described by the generic category [`:oss`](https://github.com/caskroom/homebrew-cask/blob/54a79f7dcceea9a922a5b608ac99466b9d10a191/Casks/chromium.rb#L7).
```
Comment (vitorgalvao) [inline doc/CASK_LANGUAGE_REFERENCE.md]: This must remain as it was. Linking to a specific revision means the link will always point to what we want.

File: doc/CASK_LANGUAGE_REFERENCE.md [hunk 8]
```diff
@@ -393,12 +404,12 @@ causes the creation of this symlink:
 which points to a source file such as:
 
 ```bash
-/opt/homebrew-cask/Caskroom/alfred/2.1.1_227/Alfred 2.app
+/opt/homebrew-cask/Caskroom/alfred/2.8.2_431/Alfred 2.app
 ```
 
 ### Renaming the Target
 
-You can rename the target link which appears in your `~/Applications` directory by adding a `:target` key to `app`. Example (from [scala-ide.rb](https://github.com/caskroom/homebrew-cask/blob/84e8df88836a2c11657e09264bd01b96783bb0d1/Casks/scala-ide.rb#L21):
```
Comment (vitorgalvao) [inline doc/CASK_LANGUAGE_REFERENCE.md]: This must remain as it was. Linking to a specific revision means the link will always point to what we want.

File: doc/CASK_LANGUAGE_REFERENCE.md [hunk 9]
```diff
@@ -406,7 +417,7 @@ app 'eclipse/Eclipse.app', :target => 'Scala IDE.app'
 
 ### :target May Contain an Absolute Path
 
-If `:target` has a leading slash, it is interpreted as an absolute path. The containing directory for the absolute path will be created if it does not already exist. Example (from [manopen.rb](https://github.com/caskroom/homebrew-cask/blob/84e8df88836a2c11657e09264bd01b96783bb0d1/Casks/manopen.rb#L12)):
```
Comment (vitorgalvao) [inline doc/CASK_LANGUAGE_REFERENCE.md]: This must remain as it was. Linking to a specific revision means the link will always point to what we want.

File: doc/CASK_LANGUAGE_REFERENCE.md [hunk 10]
```diff
@@ -819,16 +828,16 @@ The following methods may be called to perform standard tasks:
 
 | method                                    | availability                                     | description |
 | ----------------------------------------- | ------------------------------------------------ | ----------- |
-| `plist_set(key, value)`                   | `preflight`, `postflight`, `uninstall_preflight` | set a value in the `Info.plist` file for the app bundle. Example: [`rubymine.rb`](https://github.com/caskroom/homebrew-cask/blob/c5dbc58b7c1b6290b611677882b205d702b29190/Casks/rubymine.rb#L12)
```
Comment (vitorgalvao) [inline doc/CASK_LANGUAGE_REFERENCE.md]: Again, here, do not remove the examples. They are as intended.

Your edits are also erasing the line highlight.

File: doc/CASK_LANGUAGE_REFERENCE.md [hunk 11]
```diff
@@ -819,16 +828,16 @@ The following methods may be called to perform standard tasks:
...
+| `set_permissions(paths, permissions_str)` | `preflight`, `postflight`, `uninstall_preflight` | set permissions in `paths` to `permissions_str`. Example: [`docker.rb`](../Casks/docker.rb)
+| `suppress_move_to_applications`           | `postflight`                                     | suppress a dialog asking the user to move the app to the `/Applications` folder. Example: [`github-desktop.rb`](../Casks/github-desktop.rb)
 
 `plist_set` currently has the limitation that it only operates on the bundle indicated by the first `app` stanza (and the Cask must contain an `app` stanza).
 
 `set_ownership(paths)` defaults user ownership to the current user and group ownership to `staff`. These can be changed by passing in extra options: `set_ownership(paths, user: 'user', group: 'group')`.
 
-`suppress_move_to_applications` optionally accepts a `:key` parameter for apps which use a nonstandard `defaults` key. Example: [`alfred.rb`](https://github.com/caskroom/homebrew-cask/blob/c5dbc58b7c1b6290b611677882b205d702b29190/Casks/alfred.rb).
```
Comment (vitorgalvao) [inline doc/CASK_LANGUAGE_REFERENCE.md]: This must remain as it was. Linking to a specific revision means the link will always point to what we want.

**Comment by miccal** [comment]
All fixed.

File: doc/CASK_LANGUAGE_REFERENCE.md [hunk 12]
```diff
@@ -716,14 +714,11 @@ The elements of the `:signal` array are applied in order, only if there is an ex
 
 It is better to use the least-severe signals which are sufficient to stop a process. The `KILL` signal in particular can have unwanted side-effects.
 
-An example, with commonly-used signals in ascending order of severity:
+An example, with commonly-used signals in ascending order of severity (from [switchresx.rb](https://github.com/caskroom/homebrew-cask/blob/50ef347b517a9f293b74452367b3e2798a409e1e/Casks/switchresx.rb)):
 
 ```ruby
   uninstall :signal => [
```
Comment (vitorgalvao) [inline doc/CASK_LANGUAGE_REFERENCE.md]: Again, this is just an example. It does not have to be correct according to the cask, but it needs to pass on the information.

**Comment by vitorgalvao** [comment]
And you even added some examples that weren’t there! Thank you for your dedication to the documentation.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 9cfc59144e7933afca02dc5ca814e87ce69b93dc. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**